### PR TITLE
Add Dev Actor; wrap Iroha Client; refactor

### PR DESCRIPTION
**Pull request by 0x009922**
- Improve endpoint typing by defining types explicitly instead of `impl Responder`
- Wrap Iroha Client with async `awc` client
- Implement pagination
- Handle Iroha Query errors. Return 404 in some cases (e.g. on `GET /account/{id}` if `FindError` is returned by Iroha).
- Handle actix query errors
- Dev Actor (WIP) - an optional actix actor that periodically pushes random data to Iroha. Enabled with `dev_actor` feature and `--dev-actor` CLI flag.

### CODE REVIEW

File: src/dev_actor.rs [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+            });
+
+            Box::pin(fut)
+        } else {
+            logger::warn!(
+                "While hanlding DoRandomStuff message, it turned out there work state is empty",
+            );
+            Box::pin(async {}.into_actor(self))
+        }
+    }
```
[the remark this answers is not in the document]
Comment (appetrosyan) [reply src/dev_actor.rs:162]: Not as inefficient as you might think.

File: src/web/pagination.rs [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+}
+
+mod positive_int {
+    use color_eyre::eyre::{eyre, Report, Result};
+    use serde::{de, Serialize};
+    use std::fmt;
+
+    /// Could only contain a positive (non-zero) integer
+    #[derive(Serialize, Debug, Clone, Copy, PartialEq, PartialOrd)]
+    pub struct Positive(u32);
```
[the remark this answers is not in the document]
Comment (appetrosyan) [reply src/web/pagination.rs:136]: ```std::num::NonZeroU32```
https://doc.rust-lang.org/stable/std/num/struct.NonZeroU32.html

**Review by appetrosyan — APPROVED** [review]

File: src/dev_actor.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (0x009922) [reply src/dev_actor.rs:162]: Thanks!

File: src/web/pagination.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (0x009922) [reply src/web/pagination.rs:136]: Thanks!

**Pull request merged**